livesql/binlog.go

```diff
@@ -327,6 +332,12 @@ func (b *Binlog) parseBinlogRowsEvent(event *replication.BinlogEvent) (*update,
 	return update, nil
 }
 
+// SetOutput sets the destination for the error logger.
+func (b *Binlog) SetOutput(w io.Writer) {
+	b.logwriter = w
```

Comment:
do we want to define the logwriter interface on an io.Writer?  Should we instead define a "Logger" interface and give it the functions we want to use.  In which case, we can hook up the "Logger" to a Zap (or other) logger under the hood.

For the base case, we can create a StdOutLogger that logs directly to stdout.

Reply:
Given that this is open source, I was hoping to provide an interface that's pretty flexible that anyone could use. Do you know of a widely used logging interface?

Reply:
Going to add an interface that supports `Error` and `Info`.